devices/HapticGlove/include/SenseGloveHelper.hpp

```diff
@@ -0,0 +1,326 @@
+/**
+ * @file SenseGloveHelper.hpp
+ * @authors Kourosh Darvish <kourosh.darvish@iit.it>
+ * @copyright 2021 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2021
+ */
+
+#ifndef SENSE_GLOVE_HELPER_HPP
+#define SENSE_GLOVE_HELPER_HPP
+
+#include <Eigen/Dense>
+
+// std
+#include <array>
+#include <iostream>
+#include <memory>
+#include <stdint.h>
+
+// YARP
+#include <yarp/os/LogStream.h>
+#include <yarp/sig/Vector.h>
+
+// Sense Glove
+#include <DeviceList.h>
+#include <SenseGlove.h>
+
+/**
+ * GloveControlHelper is an helper class for controlling the glove.
+ */
+namespace senseGlove {
+class SenseGloveHelper;
+    enum class ThumperCmd : unsigned int;
+    const std::string LogPrefix = "senseGlove::SenseGloveHelper::";
+const size_t PoseSize = 7; // [ position <x, y, z>, quaternion <w, x, y, z> ]
+} // namespace senseGlove
+
+enum class senseGlove::ThumperCmd : unsigned int {
+  None = 126,
+
+  /// <summary> Turn off the thumper effects. </summary>
+  TurnOff = 124,
+
+  /// <summary> A 5-second long, constant vibration. </summary>
+  Cue_Game_Over = 118,
+
+  /// <summary> A double-click at 100% intensity. </summary>
+  Button_Double_100 = 10,
+  /// <summary> A double click at 60% intensity. </summary>
+  Button_Double_60 = 11,
+
+  /// <summary> Simulates an impact of the hand at 100% intensity. </summary>
+  Impact_Thump_100 = 1,
+  /// <summary> Simulates an impact of the hand at 30% intensity. </summary>
+  Impact_Thump_30 = 3,
+  /// <summary> Simulates an sharp impact of the hand at 40% intensity.
+  /// </summary>
+  Impact_Thump_10 = 6,
+
+  /// <summary> A light vibration to cue the user that an object it picked up.
+  /// 100% intensity. </summary>
+  Object_Grasp_100 = 7,
+  /// <summary> A light vibration to cue the user that an object it picked up.
+  /// 60% intensity. </summary>
+  Object_Grasp_60 = 8,
+  /// <summary> A light vibration to cue the user that an object it picked up.
+  /// 30% intensity. </summary>
+  Object_Grasp_30 = 9
+};
+
+class senseGlove::SenseGloveHelper {
```

Comment:
@kouroshD also, if I recall this bracket should go in a new line with the current configuration (top-level folder of the repo), make sure your clang-format picks it up

Reply:
I have tries to move the bracket in a new line, and it changed to the same line as you can see here. So, I would say that is the clang-format. Let me know if I should do, sth else?